docs/content/dependencies/cassandra-deep-storage.md

```diff
@@ -29,7 +29,7 @@ title: "Cassandra Deep Storage"
 Druid can use Cassandra as a deep storage mechanism. Segments and their metadata are stored in Cassandra in two tables:
 `index_storage` and `descriptor_storage`.  Underneath the hood, the Cassandra integration leverages Astyanax.  The
 index storage table is a [Chunked Object](https://github.com/Netflix/astyanax/wiki/Chunked-Object-Store) repository. It contains
-compressed segments for distribution to historical nodes.  Since segments can be large, the Chunked Object storage allows the integration to multi-thread
+compressed segments for distribution to Historical nodes.  Since segments can be large, the Chunked Object storage allows the integration to multi-thread
```

Comment:
I think we should delete this page. I'm pretty sure C* doesn't even work as a deep storage.

Reply:
IMO if we're going to delete this page we should delete the extension too. So it's not relevant to this patch.